src/features/comments.ts

```diff
@@ -0,0 +1,91 @@
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { commentsService } from '../api/comments';
+import { Comment } from '../types/Comment';
+import { CommentsState } from '../types/Reducer';
+
+const initialState: CommentsState = {
+  comments: [] as Comment[],
+  hasError: false,
+  loaded: true,
+  visibleForm: false,
+  submitting: false,
+};
+
+export const initComments = createAsyncThunk(
+  'comments/fetch',
+  (postId: number) => commentsService.get(postId),
+);
+
+export const initCommentDeleted = createAsyncThunk(
```

Comment:
no need to name every action starting with init
It can be just `fetchComments`, `deleteComment`, `createComment`

Same in other slices